config/Version.Public.xcconfig

```diff
@@ -1,2 +1,2 @@
-VERSION_LONG = 4.55.0.2
+VERSION_LONG = 4.55.0.1
```

Comment:
This will allow us to retry the beta build from our Rv2 GUI instead of invalidating the previous one and start anew.